spec.html

```diff
@@ -122,6 +122,32 @@ <h1>
 		</emu-note>
 	</emu-clause>
 
+	<emu-clause id="sec-create-type-error-copy" type="abstract operation">
+		<h1>
+			CreateTypeErrorCopy (
+				_realmRecord_: a Realm Record,
+				_originalError_: an ECMAScript language value
+			)
+		</h1>
+		<emu-alg>
+			1. Let _newError_ be a newly created *TypeError* object.
+			1. NOTE: _newError_ is created in _realmRecord_.
+			1. If Type(_originalError_) is Object, then
+				1. Let _message_ be Completion(Get(_originalError_, "message")).
+				1. If _message_ is not an abrupt completion, then
+					1. Let _msgString_ be Completion(ToString(_message_)).
+					1. If _msgString_ is not an abrupt completion, then
+						1. NOTE: Copying the string message when possible.
+						1. Perform CreateNonEnumerableDataPropertyOrThrow(_newError_, "message", _msgString_).
+				1. Let _cause_ be Completion(Get(_originalError_, "cause")).
```

Comment:
I'm fine ignoring the `cause` all together, but seems nice if it happens to be a string value.

Reply:
it's likely to be another error, which might have its own message.

Reply:
Yeah, I think this is a good reason to ignore the cause entirely.